src/library/scala/collection/BitSetLike.scala

```diff
@@ -240,6 +241,23 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
     }
     sb append end
   }
+  override def hashCode(): Int = {
+    val hashIterator = new AbstractIterator[Any] {
+      private var current = 0
+      private val end = nwords * WordLength
+      def hasNext: Boolean = {
+        while (current != end && !self.contains(current)) current += 1
+        current != end
+      }
+      def next(): Any =
+        if (hasNext) { val r = current; current += 1; nextHash = r.##; this }
+        else Iterator.empty.next()
+      var nextHash: Int = 0
+      override def hashCode(): Int = nextHash
```

Comment:
this is confusing to me, why do we need the hashCode of the iterator? do we actually use that?

It seems like unorderedHash doing the work, no? Can we add a comment to make this easier to read?